modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala

```diff
@@ -109,4 +114,66 @@ object ConstructionServiceSuite extends MutableIOSuite with Checkers with Transa
         expect.eql(Left(MalformedTransaction), _)
       )
   }
+
+  test("returns the accountIdentifiers for negative operations") { res =>
+    implicit val (sp, k) = res
+
+    val cs = ConstructionService.make[IO]()
+
+    val gen = for {
+      address <- addressGen
+    } yield address
```

Comment:
for comprehension is a syntax sugar for flatMap.
```
val gen = for {
  address <- addressGen
  } yield address
```
is equal to
```
val gen = addressGen
```

so there is no point to write a for comp. for one operation.

anyway, if you have more complex case, you can use both `<-` and `=` as follows:
```scala
for {
 lorem1 <- ipsum1()
 lorem2 <- ipsum2()
 lorem3 = ipsum3()
 } yield (lorem1, lorem2, lorem3)
```
is equal to:
```
ipsum1().flatMap { lorem1 =>
  ipsum2().map { lorem2 =>
    val lorem3 = ipsum3()
    
    (lorem1, lorem2, lorem3)
  }
}
```

Reply:
Removed the unnecessary `for` comprehension